### ISSUE TITLE
@angular-devkit/core is required

### ISSUE THREAD

**Issue by snebjorn** [just-jeb/angular-builders#426]
**Describe the bug**
Without `@angular-devkit/core` running `ng test` will fail with below error. Adding this package is not described in the installation guide.

```
[error] TypeError: core_1.experimental.workspace.Workspace.fromPath is not a function
    at omitted\node_modules\@angular-builders\jest\dist\index.js:26:73
    at Generator.next (<anonymous>)
    at omitted\node_modules\@angular-builders\jest\dist\index.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (omitted\node_modules\@angular-builders\jest\dist\index.js:3:12)
    at getRoots (omitted\node_modules\@angular-builders\jest\dist\index.js:22:12)
    at omitted\node_modules\@angular-builders\jest\dist\index.js:48:58
    at Generator.next (<anonymous>)
    at omitted\node_modules\@angular-builders\jest\dist\index.js:7:71
    at new Promise (<anonymous>)
```

**To Reproduce**
1. ng new test-project
2. follow install guide
3. ng test

**Expected behavior**
No error

**Builder:**
 - The name of the builder: @angular-builders/jest
 - The version of the builder: 8.0.4

**Additional context**
```
Angular CLI: 8.2.0
Node: 12.4.0
OS: win32 x64
Angular: 8.2.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.802.0
@angular-devkit/build-angular     0.802.0
@angular-devkit/build-optimizer   0.802.0
@angular-devkit/build-webpack     0.802.0
@angular-devkit/core              8.2.0 (cli-only)    <---- this is the problem
@angular-devkit/schematics        8.2.0
@angular/cdk                      8.1.2
@ngtools/webpack                  8.2.0
@schematics/angular               8.2.0
@schematics/update                0.802.0
rxjs                              6.5.2
typescript                        3.5.3
webpack                           4.38.0
```
**Fix**
`npm i -D @angular-devkit/core`

**Comment by just-jeb** [comment]
`@angular-devkit/core` is installed as a transitive dependency of `@angular/cli`. You don't have to install it explicitly.   
Are you saying that after generation a new app and installing `@angular-builders/jest` you're unable to run it because of this error?

**Comment by snebjorn** [comment]
Yep, that's exactly what I'm saying.

Note in the package list it says _(cli-only)_
```
 @angular-devkit/core              8.2.0 (cli-only)
```
That probably has something to do with it.

Are you not able to reproduce the error?

**Comment by marcalj** [comment]
Same problem with ng-cli 8.2.1

**Comment by just-jeb** [comment]
What package manager you're using guys?

**Comment by snebjorn** [comment]
Currently NPM v6.11.2. It might have been v6.11.1 when I posted, not entirely sure

**Comment by arturovt** [comment]
@snebjorn 

I could not reproduce error, see:

![image](https://user-images.githubusercontent.com/7337691/68531282-6c0bcd80-0319-11ea-9522-19c65dd39548.png)

See also this [SO](https://stackoverflow.com/questions/57188997/after-upgrading-to-angular-8-testing-with-jest-broken) link.

**Comment by snebjorn** [comment]
@arturovt you have @angular/core listed in the dependencies, remove it and try again

**Comment by just-jeb** [comment]
@arturovt This error is not easily reproducible but it exists. In the next major release (9.0.0) all the required packages (core, architect etc.) will be direct dependencies of the builders and this problem will be solved. Unfortunately cannot fix it for version 8.x.x because it is a breaking change.

**Comment by snebjorn** [comment]
will be fixed with #624

**Comment by just-jeb** [comment]
Fixed in version 9 (until official release available for installation via `@angular-builders/jest@next`.
